src/components/layout/MainLayout.tsx

```diff
@@ -109,14 +114,14 @@ const MainLayout: React.FC<Props> = ({ mobileMenuContext, children }) => {
             {appName || 'Open Path HMIS'}
           </RouterLink>
           <Box display='flex' sx={{ flexGrow: 1 }}></Box>
-          {!isMobile && (
+          {!isTiny && (
             <Stack direction='row' spacing={{ md: 0.5, lg: 2 }}>
```

Comment:
Can add `sx={{alignItems: 'center'}}` to the `Stack` in order to keep the button padding unchanged from what's currently on prod

Reply:
Thanks, added